requirements.txt

```diff
@@ -4,29 +4,32 @@
 # Format:
 # PackageName==X.Y.Z          # <comma separated list of WMCore software needing the package>
 
-Cheetah==2.4.0                # wmagent,reqmgr2,reqmon
-CherryPy==17.4.0              # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
-CMSMonitoring>=0.3.4          # wmagent,reqmgr2,reqmon,reqmgr2ms
-coverage==4.5.4               # wmagent
-cx-Oracle==5.2.1              # wmagent
-dbs-client==3.13.1            # wmagent,reqmgr2,reqmon,global-workqueue
-decorator==3.4.2              # wmagent
-funcsigs==1.0.2               # wmagent
-future==0.18.2                # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
-httplib2==0.19.0              # wmagent,reqmgr2,reqmon,acdcserver,global-workqueue,reqmgr2ms
-Markdown==3.0.1               # wmagent
-mock==2.0.0                   # wmagent
-MySQL-python==1.2.5           # wmagent
-nose2==0.9.2                  # wmagent
-psutil==5.6.6                 # wmagent,reqmgr2,reqmon,global-workqueue
-pycurl==7.43.0.3              # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
-pymongo==3.10.1               # reqmgr2ms
-pyOpenSSL==18.0.0             # wmagent
-pyzmq==17.1.2                 # wmagent
-retry==0.9.1                  # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
-rucio-clients==1.23.0         # wmagent,global-workqueue,reqmgr2ms
-setuptools==39.2.0            # wmagent
-Sphinx==1.3.5                 # wmagent,reqmgr2,acdcserver,global-workqueue,reqmgr2ms
-SQLAlchemy==1.3.3             # wmagent
-stomp.py==4.1.15              # wmagent
-CMSCouchapp==1.2.10           # wmagent
+Cheetah3~=3.2.6.post2         # wmagent,reqmgr2,reqmon
+CherryPy~=17.4.0              # wmagent,wmagent-devtools,reqmgr2,reqmon,global-workqueue,reqmgr2ms
+CMSCouchapp~=1.3.4            # wmagent
+CMSMonitoring~=0.3.4          # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
+coverage~=5.4                 # wmagent,wmagent-devtools
+cx-Oracle~=7.3.0              # wmagent
+dbs3-client~=4.0.5            # wmagent,reqmgr2,reqmon,global-workqueue
+future~=0.18.2                # wmagent,wmagent-devtools,reqmgr2,reqmon,acdcserver,global-workqueue,reqmgr2ms
+gfal2-python~=1.11.0.post3    # reqmgr2ms
+httplib2~=0.19.0              # wmagent,reqmgr2,reqmon,acdcserver,global-workqueue,reqmgr2ms
+htcondor~=8.9.7               # wmagent
+Jinja2~=3.0.1                 # wmagent
+memory-profiler~=0.58.0       # wmagent-devtools
+mock~=4.0.3                   # wmagent,wmagent-devtools
+mox3~=1.1.0                   # wmagent-devtools
+mysqlclient~=2.0.3            # wmagent
+nose~=1.3.7                   # wmagent-devtools
+nose2~=0.10.0                 # wmagent-devtools
+pep8~=1.7.1                   # wmagent-devtools
+psutil~=5.8.0                 # wmagent,wmagent-devtools,reqmgr2,reqmon,global-workqueue
+pycurl~=7.43.0.3              # wmagent,reqmgr2,reqmon,global-workqueue,reqmgr2ms
```

Comment:
We should update this pycurl version if possible, and keep it in sync with dbs3-client, but we better have another issue to track some of those updates. Just saying it now to share this and make sure others can remember to get back to this :-D

Reply:
I fixed it. :)